Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -832,13 +832,15 @@ with Accelerated Networking using Azure CLI
 for more details.
 
 When *Accelerated Networking* is enabled, ``lspci`` will show a
-Mellanox ConnectX-3 or ConnectX-4 Lx NIC:
+Mellanox ConnectX NIC:
 
 .. code-block:: shell-session
 
     $ lspci | grep Ethernet
     2846:00:02.0 Ethernet controller: Mellanox Technologies MT27710 Family [ConnectX-4 Lx Virtual Function] (rev 80)
 
+XDP acceleration can only be enabled on NICs ConnectX-4 Lx and onwards.
```

Comment:
The commit title, “clarify NIC versions for XDP acceleration”, does not correspond to this change: this is not a clarification but a correction. Can you please adjust the commit title? Would you have a source to link (ideally in the commit log) for the ConnectX-3 incompatibility?

Reply:
For the source, I see that you investigated the topic in https://github.com/cilium/cilium/issues/29065. Linking to the Mellanox doc with its known issue and/or https://github.com/cilium/cilium/issues/29065 in the commit log would be good.